src/target/riscv/riscv.c

```diff
@@ -633,7 +642,7 @@ static int set_trigger(struct target *target, unsigned int idx, riscv_reg_t tdat
 		return ERROR_TARGET_RESOURCE_NOT_AVAILABLE;
 	}
 
-	riscv_reg_t tdata1_rb, tdata2_rb;
+	riscv_reg_t tdata1_rb, tdata2_rb, maskmax;
```

Comment:
Detail:

The variable `maskmax` can be declared very close to the place of the use, which improves code readability.

```suggestion
	riscv_reg_t tdata1_rb, tdata2_rb;
```

Reply:
addressed